infer/tests/build_systems/codetoanalyze/mvn/app_with_submodules/module2parent/module2/pom.xml

```diff
@@ -19,11 +19,8 @@
         <configuration>
           <source>1.8</source>
           <target>1.8</target>
+          <encoding>UTF-8</encoding>
           <showWarnings>true</showWarnings>
-          <compilerArguments>
-            <Xlint:all/>
-            <Werror/>
-          </compilerArguments>
```

Comment:
Do we know why promoting warnings as errors is required here?

Without removing this, the compilation fails with:
```
[WARNING] bootstrap class path not set in conjunction with -source 8
```
However, it is not clear how to fix in the context of having the test compiling with different version of the Java compiler.